### PR TITLE
Add test for auto sized img with aspect-ratio inside a flex box

### PR DESCRIPTION
**Pull request by cathiechen**
Hi,
This patch added tests for flex box have an auto sized img with css aspect-ratio or aspect-ratio from width and height.
The img size should respect the value of aspect-ratio.
PTAL, thanks!

### CODE REVIEW

File: css/css-sizing/aspect-ratio/flex-aspect-ratio-027.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        max-height: 100%;
+        max-width: 100%;
+        width: auto;
+        height: auto;
+    }
+</style>
+
+</head>
+<body>
+    <div><img style="aspect-ratio: 1/1;" src="./support/200x200-green.png" /></div>
```
Comment (rwlbuis) [inline css/css-sizing/aspect-ratio/flex-aspect-ratio-027.html:24]: I think the ./ should not be needed?

File: css/css-sizing/aspect-ratio/flex-aspect-ratio-028.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        max-height: 100%;
+        max-width: 100%;
+        width: auto;
+        height: auto;
+    }
+</style>
+</head>
+
+<body>
+    <div><img width="200" height="200" src="./support/200x200-green.png" /></div>
```
Comment (rwlbuis) [inline css/css-sizing/aspect-ratio/flex-aspect-ratio-028.html:24]: I think the ./ should not be needed?

File: css/css-sizing/aspect-ratio/flex-aspect-ratio-027.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    div {
+        width: 100px;
+        display: flex;
+        flex-direction: column;
+    }
+    img {
+        max-height: 100%;
+        max-width: 100%;
+        width: auto;
+        height: auto;
```
Comment (rwlbuis) [inline css/css-sizing/aspect-ratio/flex-aspect-ratio-027.html:18]: Order is usual width, then height. Can any of these be removed, for instance the auto?

**Review by rwlbuis — COMMENTED** [review]
What is the actual difference between the tests?

File: css/css-sizing/aspect-ratio/flex-aspect-ratio-027.html (hunk 1 above)
Comment (cathiechen) [reply css/css-sizing/aspect-ratio/flex-aspect-ratio-027.html:24]: Done.

File: css/css-sizing/aspect-ratio/flex-aspect-ratio-028.html (hunk 2 above)
Comment (cathiechen) [reply css/css-sizing/aspect-ratio/flex-aspect-ratio-028.html:24]: Done.

File: css/css-sizing/aspect-ratio/flex-aspect-ratio-027.html (hunk 3 above)
Comment (cathiechen) [reply css/css-sizing/aspect-ratio/flex-aspect-ratio-027.html:18]: Done. Yeah, we should remove this.

**Comment by cathiechen** [comment]
> What is the actual difference between the tests?

Hi Rob,
Thanks for the review!
flex-aspect-ratio-027.html tests CSS aspect-ratio.
flex-aspect-ratio-028.html tests aspect ratio from attributes width and height.

**Review by rwlbuis — APPROVED** [review]

**Pull request merged**